### DISCUSSION TITLE
Is there way to update watchtower container  itself?

### DISCUSSION THREAD

**Discussion by nextcloudfan** [containrrr/watchtower#1157] [Q&A] [answered]
Is there way to update watchtower container  itself?
Watchtwoer updates all my other containers => fine
But it doe not update itself....

**Comment by simonaronsson** [comment, marked as answer]
It upgrades itself by default. However, there has not been any release in the last 8 months or so.

**Comment by nextcloudfan** [comment]
Ok, just wondering: Looking at https://hub.docker.com/r/containrrr/watchtower it mentions "updated 12 days ago"

**Reply by simonaronsson** [reply]
Yeah, that's the `latest-dev` image.
